Python/optimizer_bytecodes.c

```diff
@@ -938,6 +938,9 @@ dummy_func(void) {
     }
 
     op(_CALL_ISINSTANCE, (unused, unused, instance, cls -- res)) {
+// The below define is equivalent to PyObject_TypeCheck(inst, cls)
+#define sym_IS_SUBTYPE(inst, cls) ((inst) == (cls) || PyType_IsSubtype(inst, cls))
```

Comment:
I'm not sure about this define, maybe it's fine to duplicate this logic?

Reply:
I'll let you choose. We should probably either duplicate it, or just make it a proper function in `optimizer_symbols.c`.

Reply:
I decided to duplicate it in the end since it's only used in one uop. We can always make it a function if it sees more use.